test/GGA.js

```diff
@@ -26,11 +26,11 @@ chai.use(require('@signalk/signalk-schema').chaiModule)
 const toFull = require('./toFull')
 
 describe('GGA', () => {
-
   it('Converts OK using individual parser', done => {
-    const parser = new Parser
+    const parser = new Parser()
 
     parser.on('signalk:delta', delta => {
+      // FIXME: label is a required property in the SK Spec.
       should.not.exist(delta.updates[0].source.label)
```

Comment:
Anyone remembers why we are testing **against** the presence of a source label? A source label is required in the spec.

Reply:
The parser does not know what the label of the source should be - it depends on server configuration and the server will overwrite it.

Reply:
I think the library should always return valid SignalK. Any objection to me fixing this in another PR? The server can override it later.

Reply:
Always returning valid Signal K makes sense. We don't need to override it - you can pass the needed data in Parser constructor.